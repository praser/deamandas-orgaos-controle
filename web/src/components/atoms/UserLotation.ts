import styled from 'styled-components';

import Username from './Username';

const UserLotation = styled(Username)`
  font-size: 9px;
  font-weight: 300;
  line-height: 6px;
`;

export default UserLotation;
