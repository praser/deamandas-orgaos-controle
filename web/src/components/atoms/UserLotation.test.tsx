import React from 'react';
import { render, screen } from '@testing-library/react';
import UserLotation from './UserLotation';

describe('UserLotation atomic component', () => {
  const lotation = 'lotation';
  let userLotation: HTMLElement;

  beforeEach(() => {
    render(<UserLotation>{lotation}</UserLotation>);
    userLotation = screen.getByText(lotation);
  });

  it('is expected to match the snapshot', () => {
    expect(userLotation).toMatchSnapshot();
  });
});
