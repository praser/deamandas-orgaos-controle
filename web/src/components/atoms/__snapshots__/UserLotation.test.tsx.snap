// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserLotation atomic component is expected to match the snapshot 1`] = `
<p
  class="sc-bdvvaa sc-gsDJrp wtevO eEkqHm"
>
  lotation
</p>
`;
